src/schema/messaging/subscribeReceived.ts

```diff
@@ -0,0 +1,53 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+import { unixTime } from "@/schema/definitions/unixTime";
+
+export default defineEndpoint({
+    source: "user",
+    target: "server",
+    description: "Ask the server to send events for relevant messages",
+    request: {
+        data: Type.Object({
+            source: Type.Union([
+                Type.Object({
+                    type: Type.Literal("player"),
+                    player_id: Type.String(),
+                }),
+            ]),
+            since: Type.Optional(
```

Comment:
If it's optional, what is the default behavior?

Reply:
I didn't find a way to mark a default option, so I put it in the readme.

Reply:
I've pushed commit that sets the default